templates/cli/lib/questions.js.twig

```diff
@@ -251,7 +251,12 @@ const questionsDeployFunctions = [
       })
       return choices;
     }
-  }
+  },
+  {
+    type: "input",
+    name: "override",
+    message: 'Are you sure you want to override this function variables? This can lead to loss of secrets! Type "YES" to confirm.'
```

Comment:
```suggestion
    message: 'Are you sure you want to override this function's variables? This can lead to loss of secrets! Type "YES" to confirm.'
```